### PR TITLE
Added Haxe/Java template

### PR DESCRIPTION
**Pull request by markknol**
It's missing the _project.png_ file.

### CODE REVIEW

**Comment by elsassph** [comment]
You could probably have a single run.bat receiving the build configuration (debug/release) by adding the `$(BuildConfig)`.

**Comment by markknol** [comment]
@elsassph I wish that worked. For some reason the release build outputs as "Main.jar" where the debug outputs as "Main-Debug.jar"

**Comment by elsassph** [comment]
What I meant is that you can set the run command as `Run.bat $(BuildConfig)`, then in the batch:

    if "%1"=="debug" (
        :: run debug
    )
    else (
        :: run release
    )

**Comment by markknol** [comment]
Ah yes of course!

**Comment by markknol** [comment]
Ok done

File: FlashDevelop/Bin/Debug/Projects/370 Haxe - Java Project/Project.hxproj [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <hidden path="obj" />
+  </hiddenPaths>
+  <!-- Executed before build -->
+  <preBuildCommand />
+  <!-- Executed after build -->
+  <postBuildCommand alwaysRun="False" />
+  <!-- Other project options -->
+  <options>
+    <option showHiddenPaths="False" />
+    <option testMovie="OpenDocument" />
```
Comment (elsassph) [inline FlashDevelop/Bin/Debug/Projects/370 Haxe - Java Project/Project.hxproj]: I think test movie type should be changed to "run command".

**Pull request closed without merging**